Libplanet/Blockchain/Policies/BlockPolicy.cs

```diff
@@ -36,7 +38,7 @@ public class BlockPolicy<T> : IBlockPolicy<T>
         /// <see cref="MinimumDifficulty"/>. 1024 by default.</param>
         /// <param name="difficultyBoundDivisor">Configures
         /// <see cref="DifficultyBoundDivisor"/>. 128 by default.</param>
-        /// <param name="maxTransactionsPerBlock">Configures <see cref="MaxTransactionsPerBlock"/>.
+        /// <param name="maxTransactionsPerBlock">Configures <see cref="_maxTransactionsPerBlock"/>.
```

Comment:
It is okay to reference private property?

Reply:
Fixed them!